cider/exec/operator/aggregate/CiderAggHashTable.cpp

```diff
@@ -312,7 +307,7 @@ std::vector<CiderAggHashTableEntryInfo> CiderAggHashTable::fillColsInfo() {
   std::vector<CiderAggHashTableEntryInfo> cols_info(columns_num_);
 
   size_t offset = 0, col_slot_context_index = 0;
-  bool is_key = !is_keyless_;
+  bool is_key = true;
```

Comment:
Do we always have one or more keys here no matter it is in count or avg?

Reply:
For group-by there will be one key col at least.